ScrollAnimation.js

```diff
@@ -33,7 +33,7 @@ define([
 		 * Constructor
 		 *
 		 * @param {Element} node The node that will scroll.
-		 * @param {Object} options
+		 * @param {object} options
```

Comment:
Actually, Object should be uppercase.  But conversely, there are still some uppercase String and Boolean that should be lowercase.